src/ralph/access_cards/models.py

```diff
@@ -0,0 +1,66 @@
+from dj.choices import Choice, Choices
+from dj.choices.fields import ChoiceField
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.accounts.models import RalphUser
+from ralph.lib.mixins.models import AdminAbsoluteUrlMixin, TimeStampMixin
+
+
+class AccessCardStatus(Choices):
+    _ = Choice
+
+    new = _('new')
+    in_progress = _('in progress')
+    lost = _('lost')
+    damaged = _('damaged')
+    used = _('in use')
+    free = _('free')
+    return_in_progress = _('return in progres')
+    liquidated = _('liquidated')
+
+
+class AccessCard(AdminAbsoluteUrlMixin, TimeStampMixin, models.Model):
+    visible_number = models.CharField(
+        max_length=256,
+        null=False,
+        blank=False,
+        help_text=_('Number visible on the access card')
+    )
+    system_number = models.CharField(
+        max_length=256,
+        null=False,
+        blank=False,
+        help_text=_('Internal number in the access system')
+    )
+    issue_date = models.DateField(
+        null=True,
+        blank=True,
+        help_text=_('Date of issue to the User')
+    )
+    notes = models.TextField(
+        null=True,
+        blank=True,
+        help_text=_('Optional notes')
+    )
+    user = models.ForeignKey(
+        RalphUser,
+        null=True,
+        blank=True,
+        related_name='used_access_cards',
```

Comment:
I don't see any usage of backward relation. Please consider ``+`` if you don't need.

Reply:
changed.